detox/e2e/test/autocomplete/at_mention.e2e.js

```diff
@@ -7,14 +7,17 @@
 // - Use element testID when selecting an element. Create one if none.
 // *******************************************************************
 
-import {toChannelScreen} from '@support/ui/screen';
-
+import {logoutUser, toChannelScreen} from '@support/ui/screen';
 import {Setup} from '@support/server_api';
 
 describe('Autocomplete', () => {
     let user;
     let postInput;
 
+    afterAll(async () => {
```

Comment:
Please move block after beforeEach

Reply:
I can move it, but it should not matter as long as is defined

Reply:
It's no biggie, just wanted to make the test format consistent.